rewrite-java/src/main/java/org/openrewrite/java/tree/JavaType.java

```diff
@@ -615,9 +615,9 @@ public String toString() {
         @Nullable
         List<JavaType> bounds;
 
-        public GenericTypeVariable(@Nullable Integer managedReference, String name, Variance variance, @Nullable List<JavaType> bounds) {
+        public GenericTypeVariable(@Nullable Integer managedReference, boolean wildcard, Variance variance, @Nullable List<JavaType> bounds) {
```

Comment:
This is a breaking API change, and could be additive if we'd like to deprecate `name` instead.

Reply:
I don't think we care whether it is a wildcard or not. So can we drop this altogether?

Reply:
Okay, yes, I've removed wildcards too.